app/src/main/java/io/github/droidkaigi/confsched/util/LocaleUtil.java

```diff
@@ -103,6 +103,20 @@ public static Date getDisplayDate(@NonNull Date date, Context context) {
         }
     }
 
+    public static Date getConfTimezoneCurrentDate() {
```

Comment:
I confirmed by debug log that same date print in the case of different timezone.

```
UTC: Tue Feb 16 20:43:08 UTC 2016
Japan Standard Time: Tue Feb 16 20:44:12 GMT+09:00 2016
```